benchmarks/libpcap_fuzz_both/Dockerfile

```diff
@@ -16,7 +16,7 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder@sha256:fb1a9a49752c9e504687448d1f1a048ec1e062e2e40f7e8a23e86b63ff3dad7c
 RUN apt-get update && \
-    apt-get install -y make cmake flex bison
+    apt-get install -y make cmake flex bison libdbus-1-dev
```

Comment:
@vanhauser-thc  I added the missing dependency here.

It is weird that this dependency is specified here though https://github.com/google/fuzzbench/blob/master/fuzzers/aflplusplus/fuzzer.py#L40 I don't know why that would be the case